### PR TITLE
Re-add the Cypress Dashboard id

### PR DESCRIPTION
**Pull request by NoriSte**
As soon as Cypress will accept my OSS plan request I'll re-add the video recordings

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "cy:open": "cypress open",
     "test": "npm run cy:start-server-and-test",
     "cy:start-server-and-test": "start-server-and-test serve :5000 cy:run",
-    "cy:run": "cypress run",
+    "cy:run": "cypress run --record --key 0d5ef033-250c-4367-a396-40c9b0668b18",
```
Comment (allevo) [inline package.json]: Is this a secret?

Comment (NoriSte) [reply package.json]: Yes and no but you're right, I'm working to hide it 😉

File: cypress-run.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+const {execSync} = require('child_process');
+const uploadRecordings = process.env.npm_package_config_cypressUploadRecordings;
+let command = "npx cypress run";
+if(uploadRecordings) {
+    command += ` --record --key ${process.env.CYPRESS_RECORD_KEY}`
```
Comment (allevo) [inline cypress-run.js]: According with the documentation, this is not required.
https://docs.cypress.io/guides/guides/command-line.html#cypress-run-record-key-lt-record-key-gt
We can check the existence of the env and throw an error

Comment (NoriSte) [reply cypress-run.js]: Done!

**Pull request merged**